CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] - ReleaseDate
 ### Added
+- Add support for `TIME_STYLE` environment variable.
```

Comment:
Could you add your Github handle along with the change. Just to keep all the messages consistent.